test/Misc/testFloat.swift

```diff
@@ -0,0 +1,9 @@
+// RUN: %swift -parse-stdlib -emit-assembly %s -o - | %FileCheck %s --check-prefix=CHECK-%target-cpu
+
+// test converting floating point to unsigned integer
+import Swift
+let f = Float(-123.0)
+let x = Builtin.fptoui_FPIEEE32_Int8(f._value)
+
+// CHECK-s390x: clfebr
```

Comment:
We've had problems with CHECK patterns appearing as substrings where they didn't belong, so I suggest matching the entire line using regexes. I don't know what s390x assembly looks like, but here's an x86_64 example for this code:

```swift
func test(f: Builtin.FPIEEE32) -> Builtin.Int8 {
  return Builtin.fptoui_FPIEEE32_Int8(f)
}
```

```
// CHECK: cvttss2si %{{xmm[0-9]+}}, %{{e..}}
```